packages/flutter/lib/src/widgets/view.dart

```diff
@@ -92,3 +107,12 @@ class View extends InheritedWidget {
     return result!;
   }
 }
+
+class _ViewScope extends InheritedWidget {
```

Comment:
I call these kinds of private inherited widgets "markers" (e.g. `_FocusMarker`, `_ActionsMarker`, etc.). As a name, "scope" works too, but you might consider using "marker" for consistency.

Reply:
I did a quick count: Maybe we should rename those `Marker`s to `Scope`s. Currently, we have 5 widgets in the code base that use the suffix Marker, and 13 that uses Scope.

Reply:
Works for me.